### PR TITLE
Algolia Address autocomplete

### PR DESCRIPTION
**Pull request by Lomig**
Capture d’écran 2020-02-26 à 18 40 02

### CODE REVIEW

File: app/assets/stylesheets/components/_search_bar.scss [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-  display: flex;
-  justify-content: center;
   width: 100%;
 }
 
 .search-bar-all {
   border: 1px solid $my-green;
   border-radius: 5px;
   box-shadow: none;
+  color: $black;
```
Comment (dornby) [inline app/assets/stylesheets/components/_search_bar.scss]: Wouldn't we use $my-black?

File: app/views/components/_search_bar.html.erb [hunk 2]
```diff
@@ -2,10 +2,16 @@
...
   <div class="search-bar">
-    <form action="/" id="search-form" method="get" accept-charset="utf-8">
-      <div class="search-bar-all">
+    <form action="/places" id="search-form" method="get" accept-charset="utf-8">
+      <div class="search-bar-all" data-controller="places-search">
         <%= icon("fas", "search") %>
-        <input class="search-bar-input" type="text" placeholder="Genre, Artist, Track, Address...">
+        <input class="search-bar-input"
+               type="search"
+               placeholder="Genre, Artist, Track, Address..."
```
Comment (dornby) [inline app/views/components/_search_bar.html.erb]: Maybe we should make this consistent with what we propose now which is only "address" or "location".

Comment (Lomig) [reply app/views/components/_search_bar.html.erb]: Is it what we will propose though?
Won't we add tags?

File: app/assets/stylesheets/components/_search_bar.scss (hunk 1 above)
Comment (Lomig) [reply app/assets/stylesheets/components/_search_bar.scss]: It's the algolia tropdown. Do we want it to use our colors as well?

Comment (Lomig) [reply app/assets/stylesheets/components/_search_bar.scss]: Capture d’écran 2020-02-27 à 10 30 37


Looks good.

**Pull request merged**